ts/features/bonus/cgn/utils/svgBackground.ts

```diff
@@ -0,0 +1,25 @@
+export type Point = {
+  x: number;
+  y: number;
+};
+
+const generateRandomNumberInRange = (max: number, min: number) =>
+  Math.floor(Math.random() * (max - min + 1) + min);
+
+export const generateRandomSvgMovement = (
+  steps: number,
+  pointA: Point,
+  pointB: Point
+): string => {
+  const points: ReadonlyArray<Point> = [...new Array(steps)].map(_ => ({
+    x: generateRandomNumberInRange(pointA.x, pointB.x),
+    y: generateRandomNumberInRange(pointA.y, pointB.y)
+  }));
+
+  const pointsStringified = [
+    ...points.map(p => `${p.x} ${p.y}`),
+    `${points[0].x} ${points[0].y};`
```

Comment:
If this function is invoked with `steps=0`

`generateRandomSvgMovement(0,..,..)`

it will generate an error, maybe a crash.
Could you check `points` is greater than 0 before access it?